modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -179,13 +180,60 @@ function validateInput($values)
     $db         =& Database::singleton();
     $pscid      = (isset($values['PSCID']) ? $values['PSCID'] : null);
     $visitLabel = (isset($values['visitLabel']) ? $values['visitLabel'] : null);
+    $centerID   = (isset($values['centerID']) ? $values['centerID'] : null);
     $result     = [
-                   'PSCID'     => $pscid,
-                   'visit'     => $visitLabel,
-                   'candID'    => null,
-                   'sessionID' => null,
+                   'PSCID'             => $pscid,
+                   'visit'             => $visitLabel,
+                   'centerID'          => $centerID,
+                   'candID'            => null,
+                   'sessionID'         => null,
+                   'isValidSubmission' => true,
+                   'invalidMessage'    => null,
                   ];
 
+    if (isset($result['PSCID'], $result['centerID'])) {
+        $valid_center = $db->pselectOne(
+            "
+            SELECT
+                CenterID = :center_id
+            FROM
+                candidate
+            WHERE
+                PSCID = :psc_id
+        ",
+            array(
+             "center_id" => $result['centerID'],
+             "psc_id"    => $result['PSCID'],
+            )
+        );
+        if (!$valid_center) {
+            $valid_center = $db->pselectOne(
+                "
+                SELECT
```

Comment:
I might be wrong, but I feel like checking whether a candidate with provided `PSCID` and `CenterID` is enough of validation @driusan